From c5a4b95c72dfeaf2b04b27f53d8f15b9b3aa8f09 Mon Sep 17 00:00:00 2001
From: Liu Ying <Ying.Liu@freescale.com>
Date: Tue, 26 Jun 2012 12:45:05 +0800
Subject: [PATCH 1648/2463] ENGR00214865 mxc_v4l2_capture:Be silent when
 closing device

This patch changes the debug level of a kernel message in
mxc_v4l2_close() from KERN_INFO to KERN_DEBUG to make the
console silent when closing device.

Signed-off-by: Liu Ying <Ying.Liu@freescale.com>
(cherry picked from commit 42bd808baa7c03671665297cd40ae9cce8186d6b)
---
 drivers/media/video/mxc/capture/mxc_v4l2_capture.c |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/drivers/media/video/mxc/capture/mxc_v4l2_capture.c b/drivers/media/video/mxc/capture/mxc_v4l2_capture.c
index 5bd4886..7f3b869 100644
--- a/drivers/media/video/mxc/capture/mxc_v4l2_capture.c
+++ b/drivers/media/video/mxc/capture/mxc_v4l2_capture.c
@@ -1705,7 +1705,7 @@ static int mxc_v4l_close(struct file *file)
 
 		wait_event_interruptible(cam->power_queue,
 					 cam->low_power == false);
-		pr_info("mxc_v4l_close: release resource\n");
+		pr_debug("mxc_v4l_close: release resource\n");
 
 		if (strcmp(mxc_capture_inputs[cam->current_input].name,
 			   "CSI MEM") == 0) {
-- 
1.7.7.4

